src/junit/LIC_test.java

```diff
@@ -8,4 +8,16 @@
 import static org.junit.Assert.assertTrue;
 
 public class LIC_test {
+    /*
```

Comment:
From the assessment criteria on [Canvas](https://canvas.kth.se/courses/37918/assignments/235345): 

* The contract that is tested is clearly formulated in natural language
* The assertions of the tests correctly and completely assess the contract

I think the natural language part is missing, which should define what the test is testing and expectation.